docs/getting_started.rst

```diff
@@ -0,0 +1,394 @@
+.. _getting_started:
+
+**********************
+Getting Started
+**********************
+
+.. contents:: Table of contents:
+   :local:
+
+Installation
+------------
+
+Sgkit itself contains only Python code but many of the required dependencies use compiled code or have system
+package dependencies. For this reason, conda is the preferred installation method.  There is no sgkit conda
+package yet though (or pypi package for that matter) so the recommended setup instructions are::
+
+    $ conda install -c conda-forge --file requirements.txt
+    $ pip install -e .
+
+..
+
+Overview
+--------
+
+Sgkit is a general purpose toolkit for quantitative and population genetics.
+The primary goal of sgkit is to take advantage of powerful tools in the `PyData ecosystem <https://pydata.org/>`_
+to facilitate interactive analysis of large-scale datasets. The main libraries we use are:
+
+- `Xarray <http://xarray.pydata.org/en/stable/>`_: N-D labeling for arrays and datasets
+- `Dask <https://docs.dask.org/en/latest/>`_: Parallel computing on chunked arrays
+- `Zarr <https://zarr.readthedocs.io/en/stable/>`_: Serialization for chunked arrays
+- `Numpy <https://numpy.org/doc/stable/>`_: N-D in-memory array manipulation
+- `Pandas <https://pandas.pydata.org/docs/>`_: Tabular data frame manipulation
+- `CuPy <https://docs.cupy.dev/en/stable/>`_: Numpy-like array interface for CUDA GPUs
+
+Data structures
+~~~~~~~~~~~~~~~
+
+There are currently no data models in the library that attempt to capture the complexity of many (or even common)
+analyses and the data structures that would support them -- operations are applied primarily to Xarray
+`Dataset <http://xarray.pydata.org/en/stable/data-structures.html#dataset>`_ objects instead. Users are free to manipulate data
+within these objects as they see fit, but they must do so within the confines of a set of conventions for variable
+names, dimensions, and underlying data types. The example below illustrates a ``Dataset`` format that would result
+from an assay expressible as `VCF <https://en.wikipedia.org/wiki/Variant_Call_Format>`_,
+`PLINK <https://www.cog-genomics.org/plink2>`_ or `BGEN <https://www.well.ox.ac.uk/~gav/bgen_format/>`_.
+This is a guideline however, and a ``Dataset`` seen in practice might include many more or fewer variables and dimensions.
+
+.. image:: _static/data-structures-xarray.jpg
```

Comment:
Ah good call, I made it with google drawings so I made this read-only to anyone with the link so it could be duplicated+edited: https://github.com/pystatgen/sgkit/pull/278/commits/f7f5a039412fc6b4c29621d283c1a5738b746521#diff-9bc8bf6e8d9db020cefce1fbc0426795R46-R48